pyodide_build/xbuildenv.py

```diff
@@ -404,6 +404,116 @@ def uninstall_version(self, version: str | None) -> str:
 
         return version
 
+    def clone_emscripten(self, emsdk_dir: Path | str | None = None) -> Path:
```

Comment:
Let's make this function private, to avoid using it directly outside.

```suggestion
    def _clone_emscripten(self, emsdk_dir: Path | str | None = None) -> Path:
```